lib/SILOptimizer/Utils/CanonicalizeInstruction.cpp

```diff
@@ -103,6 +110,41 @@ simplifyAndReplace(SILInstruction *inst, CanonicalizeInstruction &pass) {
 //                        Canonicalize Memory Operations
 //===----------------------------------------------------------------------===//
 
+// Replace all uses of an original struct or tuple extract instruction, with the
+// given load instruction. The caller ensures that the load only loads the
+// extracted field.
```

Comment:
You are assuming below a specific form of the code (e.x.: the cast <BeginBorrowInst>()). Can you add an example in this comment just to be /really/ explicit.

Reply:
Sure. Done.